tecton_client/_internal/request_utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import Dict
+
+
+class MetadataOptions:
+    """Passed into metadata_options on get_features, controls what metadata is returned as part of the response."""
+
+    def __init__(
+        self,
+        include_names: bool = True,
+        include_data_types: bool = True,
+        include_effective_times: bool = False,
+        include_slo_info: bool = False,
+        include_serving_status: bool = False,
+        include_all: bool = False,
```

Comment:
not a fan of this one because it doesn't reflect a real option in the API. An alternative to this would be to have a `MetadataOptions.all()` classmethod